.github/workflows/firebase_deploy.yaml

```diff
@@ -0,0 +1,22 @@
+name: Continuous deployment to firebase hosting
+
+on:
+  push: 
+    branches: [ cmd-f_dev ]
```

Comment:
are we setting this one up for `cmd-f_main` as well?

Reply:
Good point. I will make the dev branch target a site in the dev db, while the main branch will target a site in prod